draft-ietf-mls-protocol.md

```diff
@@ -3085,12 +3090,13 @@ considered invalid.
 # Group Evolution
 
 Over the lifetime of a group, its membership can change, and existing members
-might want to change their keys in order to achieve post-compromise security. In
-MLS, each such change is accomplished by a two-step process:
+might want to change their keys in order to achieve post-compromise security.
+In MLS, changes can be made immediately using a Commit message with embedded
+Proposals, such changes can be accomplished via a two-step process:
```

Comment:
```suggestion
In MLS, each such change is accomplished by a two-step process:
```

Since every change requires a proposal, stating this as Proposal/Commit first, then one-step below.